docs/tutorials/motor_noise_debugging.rst

```diff
@@ -0,0 +1,99 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _motor_noise_debugging:
+
+***********************************************
+Communication Errors From Motor Noise
+***********************************************
+
+Where does motor noise come from?
+============================
+Electric motors produce torque through interactions between the motor’s permanent magnetic field and the electric current flowing through the motor’s wire windings. 
+The electric current in the motor’s windings produces a variable magnetic field that can be controlled to exert a force on the motor’s permanent magnets/magnetic field. 
+Current is directed through the motor’s wire windings by applying a voltage across the windings. The direction the current flows through the windings can be switched by 
+changing the voltage polarity across the windings. In a three phase brushless DC motor like those produced by Vertiq, the three windings are connected between each of the 
```

Comment:
https://www.google.com/imgres?imgurl=https%3A%2F%2Fwww.electronics-tutorials.ws%2Fwp-content%2Fuploads%2F2020%2F02%2Fmag21.gif%3Ffit%3D157%252C162%3Ffit%3D157%2C226&tbnid=1tOU7WRpjECrFM&vet=12ahUKEwjNy728oOqDAxVbHGIAHas7AnsQMygQegQIARBu..i&imgrefurl=https%3A%2F%2Fwww.electronics-tutorials.ws%2Felectromagnetism%2Felectromagnetic-induction.html&docid=96GGsyn9_BSl-M&w=157&h=162&q=magnetic%20field%20coil%20current%20voltage&ved=2ahUKEwjNy728oOqDAxVbHGIAHas7AnsQMygQegQIARBu

This is a good illustration of what I'm describing. I probably have to recreate it because I have no idea if the image is copy righted. Or were you asking for more of a motor diagram? I'm in a little bit of a hard place because I want to keep things general enough to apply to all electric motors instead of any one specific topology.

Reply:
Added a diagram. The idea of keeping things generalized is that commutation noise exists on other electric motors and I don't want that to come across as an "us problem". Later in the document I explain why vehicles using our motors can be more sensitive to it. The intention of the article is to explain where the noise/interference is coming from and provide strategies to prevent it.